template/.eslintrc

```diff
@@ -1,4 +1,8 @@
 {
+  "//": [
+    "If unexplained eslint errors appear in the log",
+    "be sure to check if @babel/eslint-parser is installed."
+  ],
```

Comment:
Is this necessary since @babel/eslint-parser was added to the dependencies?

Reply:
Also see my comment on https://github.com/AlaskaAirlines/WC-Generator/issues/266#issuecomment-967239916: are we sure adding this dependency is the right solution?

Reply:
@jason-capsule42 I put the note there in case someone manually removes the IE dependencies and then things break. 

@geoffrich the issue that I ran into was with stylelint. The solution here is what I found to fix it. If the update you suggest work, I'm all for doing that. I have a repo in play where I can test that.

Reply:
@blackfalcon try it and let me know what the error is if you do run into it. If we aren't babel-izing code, I don't think we should need to add a babel dep.